DOCS-2336 and DOCS-2369 Synthetics CI/CD Testing Edits (#11447)
DataDog/documentation: content/en/synthetics/api_tests/icmp_tests.md

ICMP tests can run from [managed][1] and [private locations][2] depending on whether you want to trigger ICMP pings to your endpoints from outside or inside your network. You can run ICMP tests on a defined schedule, on demand, or from within your [CI/CD pipelines][3].

## Permissions

By default, only users with the [Datadog Admin and Datadog Standard roles][4] can create, edit, and delete Synthetic ICMP tests. To get create, edit, and delete access to Synthetic ICMP tests, upgrade your user to one of these two [default roles][4].

If you have access to the [custom role feature][5], add your user to a custom role that includes `synthetics_read` and `synthetics_write` permissions for Synthetic Monitoring.

## Configuration

> Now that you have chosen to create an [`ICMP` test][6], define your test's request.

Reviewer: ```suggestion
After choosing to create an [`ICMP` test][6], define your test's request.
```
